.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+---
+contact_links:
+  - name: Ask a question or get support
+    url: https://trinodb.slack.com/channels/python-client
```

Comment:
trinodb/trino is trying to use GitHub Discussions - much less friction. For problems which require more interactive troubleshooting we generally just ask people to continue on the Slack.

As a user if you asked me to create an account, click link in email, then add some personal details before being able to ask a question I'd probably not ask it and may decide to not use it altogether.

Reply:
We can add the GitHub Discussions as a first option as well and distinguish accordingly.

Reply:
@martint Can you enable discussions for this repo as well?

Reply:
Done